docs/qa/CheckLists/FavouritesPage.md

```diff
@@ -0,0 +1,91 @@
+| 38 | From top to bottom 1) Selected "x" where x is a number of selected tags 2) "Select all tags" button, 3) statistics on how many tags and words are presented 4) Tags list                                                                                        |             |                  |
+| 39 | Pressing "Select all tags" button when it is not activated will select all tags                                                                                                                                                                                 |             |                  |
+| 40 | Pressing it again will unselect all tags                                                                                                                                                                                                                        |             |                  |
+| 41 | If the tag is chosen, it will be highlighted by dark blue background and a checkmark to the left of a tag will appear                                                                                                                                           |             |                  |
+|    | 16 delete                                                                                                                                                                                                                                                       |             | Design page 16   |
+| 42 | Check all UI texts/elements on design page 16                                                                                                                                                                                                                   |             |                  |
+| 43 | Pressing "Delete" button will put all selected tags to bin                                                                                                                                                                                                      |             |                  |
+| 44 | Swipe down or pressing outside bottom sheet will close it                                                                                                                                                                                                       |             |                  |
+| 45 | If it was closed, all tags are still selected, none tags deleted                                                                                                                                                                                                |             |                  |
+| 46 | If "Delete button" was pressed, bottom sheet goes down, deleted tags will disappear from list                                                                                                                                                                   |             |                  |
```

Comment:
Only the name of button should be in the quotes

Reply:
I removed 'button' from quotes, added 'Button" after quotes